README.md

```diff
@@ -20,11 +20,7 @@ This Lowdefy application consists of four pages
     - You can read more about the [Lowdefy MongoDB connector](https://docs.lowdefy.com/MongoDB).
 - Clone this repository.
 - Create a `.env` file in your project folder and set your MongoDB database connector URI as a variable in the `.env` file: `LOWDEFY_SECRET_EXAMPLES_MDB="{{ your_mongodb_connection_uri }}"`
-- In the command console, navigate to your project folder and run the Lowdefy CLI: `npx lowdefy@latest dev`.
-
-## Deploying this example
-
-This simplest solution is to deploy this example to [Netlify](https://netlify.com). See [the Netlify Deployment instructions](https://docs.lowdefy.com/deployment) for more detail on how to deploy a Lowdefy app to Netlify.
+- In the command console, navigate to your project folder and run the Lowdefy CLI: `pnpx lowdefy@4.0.0-alpha.37 dev`.
```

Comment:
`pnpx lowdefy@4 dev`

Reply:
Change made in 7f192eefabd533754b4ff456e037b40f4b5206a2